src/sparseml/pytorch/sparsification/pruning/modifier_pruning_mfac.py

```diff
@@ -333,9 +333,13 @@ def _collect_grad_samples(
         )
 
         _LOGGER.debug("Starting to collect {num_grads} grads with GradSampler")
+        _LOGGER.info("Setting the model in the eval mode")
```

Comment:
Status updates should be logged in debug
```suggestion
        _LOGGER.debug("Setting the model in the eval mode")
```

Reply:
Done